### DISCUSSION TITLE
Image conversion/management appears to be broken?

### DISCUSSION THREAD

**Discussion by mlawren** [PostOwl/postowl#46] [General]
I'm running PostOwl from source, using `npm run dev`, on Debian bookworm, and accessing localhost:5173 directly.

From the urls it looks like you are converting jpg/png images into webm for presentation. However what I'm seeing appears broken:

![image](https://github.com/PostOwl/postowl/assets/82904/a299999a-df4f-49b0-9d62-5c47022f022f)

It could be that my Firefox 115.0.2 on Windows 8.1 is too old or unsupported? Otherwise there might be something else going on.

**Comment by keybits** [comment]
We convert images to webp on the client before uploading. As far as I'm aware this should be supported by Firefox 115.

Are you able to try on a more modern OS in case Windows 8 is causing issues? We've not had any issues on Windows 10, macOS or recent Ubuntu Gnome.

**Reply by mlawren** [reply]
Ah, that makes it easier to test.  I don't have a Windows >= 8.1, but the latest working Chrome for this operating system does appear to do the encoding correctly. So we can chalk this up to client-side error.

[Chrome here has an out-of-memory issue with PostOwl, but I don't complicate any more discussions on out-of-date issues.]

**Reply by keybits** [reply]
Thanks for letting us know. Safari on macOS also has some issues with the client side image conversion so we might look into a more robust solution.

The current solution is nice because it creates small images (100kb approx) for inserting into the database and prevents accidental upload of huge files.
